HULKs/hulk
# Check OS-Version

## Introduced Changes
Prevent upload in case of unexpected os-version.  This check can be disabled by using '--skip-check-os'.

Fixes #35 

## ToDo / Known Issues

## Ideas for Next Iterations (Not This PR)


## How to Test

pepsi upload #nao-number
and
pepsi upload #nao-number --skip-check-os 

The first command should fail if nao's os-version doesn't match
the expected version and the execution can be forced by the second command.
